t/01-customers.t

```diff
@@ -18,7 +18,7 @@ subtest 'basic stuff' => sub {
 };
 
 subtest 'create customer with no data' => sub {
-    my $cust = stripe->create_customer;
+    my $cust = stripe->create_customer({});
```

Comment:
I think I had to make the same change in my PR, and IMO this is okay. (It's opinionated, but I think the client should encourage passing at least some data to the Stripe API.)